java-client/session/src/main/java/io/deephaven/client/impl/TableHandleManagerSerial.java

```diff
@@ -21,6 +21,7 @@
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
```

Comment:
wrong patch? changes in this file don't look like they belong here either, unless java 11-17's type inference got a lot worse?

Reply:
This method import can go away, but the changes I made to the lambdas (w/ type hints) seem to be necessary.